views/models/UsersModel/index.js

```diff
@@ -13,6 +13,24 @@ const UsersModel = {
       isFixed: reader === owner,
     })),
   createReadersList: (readers) => readers.map(({ value }) => value),
+  createUsersAccesList: (access) =>
+    access.map((site) => ({
+      label: site,
+      value: site,
+    })),
+  createUserFromFormValues: (user) => ({
+    ...user,
+    role: user.role?.value || user.role,
+    access: user.access.map(({ value }) => value),
+  }),
+  createFormValuesFromUser: (user) => ({
+    ...user,
+    role: { value: user.role },
+    access: user.access.map((site) => ({
+      label: site,
+      value: site,
+    })),
```

Comment:
can we use the model method here?

```suggestion
    access: UserModel.userAccessDropdownOptions(user.access),
```